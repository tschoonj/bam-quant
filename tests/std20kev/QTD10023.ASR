File: C:\xrfdaten\2014KW30\chris\std20kev\std10023.spe
Start: 21.07.2014 18:01:47
Stop: 21.07.2014 18:03:27
BESSY: 298.8487675 298.8781506
IONI: 3.979217E-11 3.955165E-11
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 91887
Run: 100000
Dead: 8
ICR: 22480
OCR: 0
Accumulated BESSY: 59712.4739339
Accumulated Ioni: 7.90850552000001E-09
N Accumulation: 200
Normfactor:      0.27522096
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: V
exp_x 91.2603988647461 91.2604
exp_y 150.323791503906 150.3238
exp_z 53.8899993896484 53.89
exp_w 135 135
exp_Tisch 870.6 870.600001928
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 20 20
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 3.00000000000001E-02
$PEAKS:   
      1
  23   1      4.950   1731473.      1316.      72.27
