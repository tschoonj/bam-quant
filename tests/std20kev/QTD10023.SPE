$DATA:
0 2000
21567
13758
5813
1631
330
61
8
2
1
1
1
0
0
0
0
1
98
536
579
624
587
567
573
521
504
468
477
451
451
470
454
432
416
387
401
381
373
369
416
383
377
357
363
357
332
354
349
308
313
345
316
328
302
338
298
285
312
312
331
270
319
309
336
322
304
280
291
313
295
231
282
297
286
315
276
290
267
287
283
292
262
260
268
247
254
254
260
246
272
261
297
256
262
314
266
268
268
311
312
297
284
366
334
307
303
308
308
309
280
297
282
287
308
286
298
272
274
279
304
295
263
289
267
280
272
278
314
295
301
304
295
324
299
303
318
295
291
278
289
308
330
339
336
383
431
498
521
601
490
437
414
398
426
499
677
930
1361
1638
1906
1935
1621
1282
875
636
474
378
340
365
318
344
335
312
330
371
327
346
359
364
388
468
518
566
587
532
529
457
418
412
375
398
383
405
419
453
413
436
454
431
457
476
445
467
506
466
494
499
515
534
540
533
522
546
636
598
586
624
571
621
628
585
648
688
665
668
731
764
784
814
858
936
1004
1103
1291
1548
1822
2471
3666
6688
13192
27288
52037
89636
138398
189191
228750
245762
229062
189397
136652
88320
49474
25071
11338
4940
2308
1379
1173
1144
1210
1466
1955
2866
4663
7645
12616
18963
26437
32982
37234
37196
33049
26106
18351
11476
6300
3193
1448
556
209
83
47
21
20
13
13
14
21
14
10
13
15
15
13
14
10
5
11
15
14
13
15
11
18
11
14
13
5
10
9
8
8
9
6
18
18
19
33
29
16
15
17
14
16
5
7
4
12
3
5
11
11
11
8
5
10
5
8
5
9
7
9
6
8
7
13
7
10
12
16
13
10
5
8
8
12
6
9
11
6
8
7
7
6
11
7
5
8
19
11
21
8
10
12
4
7
9
7
5
8
12
12
2
8
7
8
5
9
10
8
9
12
7
5
11
11
16
15
9
13
12
16
11
14
12
21
14
13
16
8
15
15
10
8
7
10
10
10
18
11
8
9
5
12
9
15
7
12
22
9
12
18
7
12
7
12
10
10
9
6
7
14
5
7
9
10
11
7
12
10
8
13
6
7
7
10
14
11
9
7
15
16
10
9
12
9
8
11
9
7
6
12
15
15
13
15
16
18
8
22
20
29
39
61
80
120
194
228
257
296
387
330
349
281
209
202
116
88
42
41
17
14
12
11
16
23
28
35
32
54
52
77
96
99
100
119
105
106
80
66
45
27
18
14
13
15
7
5
14
7
6
14
12
6
13
14
15
17
16
12
14
11
7
10
8
9
11
5
6
12
3
4
8
6
4
8
9
6
5
10
10
8
2
6
2
6
3
5
7
2
4
9
6
10
7
2
3
2
7
5
3
8
6
5
8
5
5
8
6
4
7
5
1
5
3
0
6
5
6
9
3
0
7
4
3
1
7
4
9
4
4
1
4
6
6
4
9
3
4
9
4
5
4
4
6
5
4
7
4
7
8
6
4
4
3
4
1
5
6
5
2
4
4
1
5
3
3
3
3
5
5
4
3
2
2
1
3
9
6
3
4
1
2
6
3
5
6
3
3
2
3
3
6
7
2
1
3
5
7
8
3
3
2
3
1
0
2
5
2
4
3
3
8
2
6
2
3
5
3
4
0
7
8
3
3
1
2
1
2
6
7
8
3
2
1
3
4
3
3
1
2
3
2
0
3
1
5
3
4
6
6
2
1
3
4
0
3
3
3
1
0
2
1
0
2
2
1
2
3
4
3
1
5
2
3
4
3
4
2
0
2
4
1
3
1
1
3
6
2
4
6
12
8
21
29
32
18
22
43
32
47
35
38
40
31
21
24
18
14
9
7
3
4
8
5
2
7
2
4
4
2
2
6
3
4
1
4
2
8
12
15
15
15
40
58
62
78
96
111
123
125
137
153
138
124
114
111
95
65
67
36
30
31
18
12
8
5
2
3
5
3
2
1
4
4
7
6
5
3
7
7
10
4
8
15
21
24
21
33
45
48
67
69
61
77
73
70
54
71
47
56
52
34
27
29
17
20
16
15
11
13
11
4
13
8
7
10
9
6
10
6
15
5
6
12
9
7
13
12
8
8
8
10
12
5
9
12
14
9
9
9
17
17
11
20
25
18
37
37
25
29
55
40
41
37
38
44
40
34
27
32
30
30
30
36
39
41
33
37
64
45
55
55
47
45
46
59
57
59
68
60
55
55
58
53
71
69
64
54
61
65
81
67
75
89
86
89
70
79
95
71
89
81
87
95
81
95
108
126
111
103
117
122
155
168
183
188
187
203
243
176
191
217
187
193
184
167
161
143
129
117
115
113
100
57
59
48
35
34
34
19
19
7
12
3
10
4
2
1
3
3
2
1
1
0
1
0
1
1
0
1
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
1
0
1
0
2
2
1
1
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
1
0
0
0
0
1
0
1
0
0
1
1
0
0
0
0
0
0
0
0
0
0
0
0
0
2
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
2
0
0
0
0
0
0
1
0
0
0
0
0
0
0
1
0
0
0
0
2
0
1
0
0
0
0
0
0
1
1
1
0
1
2
1
0
2
2
1
0
0
1
1
0
2
0
0
1
2
4
0
1
0
0
0
1
0
0
0
3
0
0
0
0
2
1
0
0
1
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
File: C:\xrfdaten\2014KW30\chris\std20kev\std10023.spe
Start: 21.07.2014 18:01:47
Stop: 21.07.2014 18:03:27
BESSY: 298.8487675 298.8781506
IONI: 3.979217E-11 3.955165E-11
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 91887
Run: 100000
Dead: 8
ICR: 22480
OCR: 0
Accumulated BESSY: 59712.4739339
Accumulated Ioni: 7.90850552000001E-09
N Accumulation: 200
Normfactor:      0.27522096
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: V
 
exp_x 91.2603988647461 91.2604
exp_y 150.323791503906 150.3238
exp_z 53.8899993896484 53.89
exp_w 135 135
exp_Tisch 870.6 870.600001928
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 20 20
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 3.00000000000001E-02
$PEAKS:   
      1
  23   1      4.950   1731473.      1316.      72.27
