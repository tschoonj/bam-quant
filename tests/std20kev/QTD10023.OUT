     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  AXIL IBM-PC V3.00                       10-08-2014   14:23:37  บ   
     บ  Spectrum: QTD10023.SPE                                     0.s บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  Fitting Region: channels 100 -1980        4 iterations done    บ   
     บ  ChiSquare =    47.3   last change =   .03%    lambda= 1.E-06   บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                        CALIBRATION  DATA                        บ   
     ฬออออออออออออออออหออออออออออออออออออออออออหอออออออออออออออออออออออน   
     บ                บ   Initial estimate     บ    Final estimate     บ   
     ฬออออออออออออออออฮออออออออออออออออออออออออฮอออออออออออออออออออออออน   
     บ  ZERO (eV)     บ      -51.0 ๑   100.0   บ     -51.0 ๑     1.0   บ   
     บ  GAIN (eV/ch)  บ     20.333 ๑   2.000   บ    20.333 ๑    .004   บ   
     บ det NOISE (eV) บ      -71.3 ๑    40.0   บ     -71.3 ๑    -5.7   บ   
     บ  FANO factor   บ       .228 ๑    .050   บ      .228 ๑    .005   บ   
     ศออออออออออออออออสออออออออออออออออออออออออสอออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                           PEAK  DATA                            บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ  # Line       E(KeV)  rel. int.   peak area   st. dev    chi-sq บ   
     บ               chan#   fwhm (eV)      backgr            tot. abs บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                                                                 บ   
     บ  1 V -Ka                           1731473. ๑   1316.           บ   
     บ    KA1         4.950   1.00000     1731473. ๑   1316.     72.3  บ   
     บ              245.951    137.75         147.           8.31E-01  บ   
     บ                                                                 บ   
     บ  2 V -Kb                            282639. ๑    532.           บ   
     บ    KB1         5.427   1.00000      282639. ๑    532.     47.3  บ   
     บ              269.410    145.93         161.           8.69E-01  บ   
     บ                                                                 บ   
     บ  3 Pile_up                            4095. ๑     67.           บ   
     บ    L1N3        9.900    .73888        3026. ๑     51.     17.7  บ   
     บ              489.393    207.46         111.           9.76E-01  บ   
     บ    L1N2       10.377    .24140         989. ๑     19.      6.4  บ   
     บ              512.852    212.98         103.           9.79E-01  บ   
     บ    L1O3       10.854    .01972          81. ๑     10.      1.0  บ   
     บ              536.311    218.35         106.           9.82E-01  บ   
     บ                                                                 บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                         FILTER BACKGROUND                       บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
